apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -107,6 +113,17 @@ const AvailableTimes: FC<AvailableTimesProps> = ({
                     )}
                     data-testid="time">
                     {slot.time.format(timeFormat)}
+                    {seatsPerTimeSlot && (
+                      <p
+                        className={`${
+                          slot.attendees && slot.attendees / seatsPerTimeSlot >= 0.5
+                            ? "text-rose-600"
+                            : "text-emerald-400"
+                        } text-sm`}>
+                        {slot.attendees ? seatsPerTimeSlot - slot.attendees : seatsPerTimeSlot} /{" "}
+                        {seatsPerTimeSlot} Seats available
+                      </p>
+                    )}
```

Comment:
Even if the booking is full we still want to to render the time. TODO need to disable times if there are no more seats